src/index.js

```diff
@@ -5,63 +5,108 @@ import {
   Route,
   Switch,
 } from 'react-router-dom';
+import { FormattedMessage } from 'react-intl';
 
+import { AppContextMenu } from '@folio/stripes/core';
 import {
   checkScope,
   CommandList,
   defaultKeyboardShortcuts,
   HasCommand,
+  NavList,
+  NavListItem,
+  NavListSection,
 } from '@folio/stripes/components';
 
+import {
+  AcqKeyboardShortcutsModal,
+  useModalToggle,
+} from '@folio/stripes-acq-components';
+
 import Main from './components/Main';
 import Settings from './settings';
 
-class Finance extends React.Component {
-  static propTypes = {
-    match: PropTypes.object,
-    showSettings: PropTypes.bool,
-    stripes: PropTypes.object,
-    history: PropTypes.object,
-  }
-
-  focusSearchField = () => {
+const Finance = ({
+  match,
+  showSettings,
+  location,
+}) => {
```

Comment:
Now that it's a functional component, could we use [router hooks](https://reactrouter.com/web/api/Hooks) to get `match` and  `location`?
Not to drill it down the props